templates/email/components/layout-start.ejs

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:o="urn:schemas-microsoft-com:office:office" lang="en" style="font-family:arial, 'helvetica neue', helvetica, sans-serif">
+
+<head>
+    <meta charset="utf-8"/>
```

Comment:
This line gave me a little headache. The original one was `<meta charset="utf-8">`, without the trailing slash. Somehow, when getting the emails from the mocked server, the version with final slash was returned, causing some tests to break due to rendered version not matching the expected version.